arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -185,5 +185,47 @@ public async Task GetGraphEdgeCollectionsAsync_ShouldThrow_WhenGraphIsNotFound()
             Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
             Assert.Equal(1924, exception.ApiError.ErrorNum); // GRAPH_NOT_FOUND
         }
+
+        [Fact]
+        public async Task PostGraphEdgeAsync_ShouldSucceed()
+        {
+            string tempGraph = nameof(PostGraphEdgeAsync_ShouldSucceed);
+            var postEdgeGraph = await _client.PostGraph(new PostGraphBody
+            {
+                Name = tempGraph,
+                EdgeDefinitions = new List<EdgeDefinition>()
+            });
+            var response = await _client.PostGraphEdgeDefinitionAsync(tempGraph, new EdgeDefinition
+            {
+                From = new string[] { "fromclxx" },
+                To = new string[] { "toclxx" },
+                Collection = "clxx"
+            });
+            Assert.Equal(HttpStatusCode.Accepted, response.Code);
+            Assert.False(response.Error);
+            Assert.Single(response.Graph.EdgeDefinitions);
+            Assert.Equal(tempGraph, response.Graph.Name);
+            Assert.Equal("_graphs/" + tempGraph, response.Graph._id);
+            Assert.NotNull(response.Graph._rev);
+            Assert.False(response.Graph.IsSmart);
+            Assert.Empty(response.Graph.OrphanCollections);
+        }
+
+        [Fact]
+        public async Task PostGraphEdgeAsync_ShouldThrow_WhenNotFound()
```

Comment:
Name of test method should be updated to read `PostGraphEdgeDefinitionAsync_ShouldThrow_WhenNotFound`. Also maybe give a more specific `When`? Like: `PostGraphEdgeDefinitionAsync_ShouldThrow_WhenGraphNotFound`